benchmark-sets/jvm-all/antlr3-java.yaml

```diff
@@ -0,0 +1,55 @@
+"functions":
+- "exceptions": []
```

Comment:
Is `exceptions` a new field in FI?
I noticed this new field in the new C/C++ benchmarks, too.
It is harmless to OFG, but if it serves no actual purpose, removing it would be cleaner.

Reply:
Nice catch. It's used but we need to bump introspector for this to reflect in the benchmarks generated from introspector.oss-fuzz.com versus locally. Doing this before landing this PR by way of https://github.com/google/oss-fuzz/pull/12170

Reply:
The benchmarks have been updated now! Will run a small experiment and if all goes well there then I will run a large experiment